names/attr_constants.csv

```diff
@@ -227,3 +227,4 @@ vpc_endpoint_id,VPCEndpointID
 vpc_id,VPCID
 vpc_security_group_ids,VPCSecurityGroupIDs
 weight,Weight
+workspace_id,WorkspaceID
```

Comment:
Added this names constant, but it triggers a refactor for all the places where there are string litterals with "workspace_id" https://github.com/hashicorp/terraform-provider-aws/actions/runs/9664687528/job/26659952436?pr=38101

Should I go ahead and use `names.AttrWorkspaceID` or should I revert this commit instead?